.github/workflows/ports_windows.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           runner: windows-latest
           vs_version: '[15, 16)'
         - visualstudio: '2019'
-          runner: windows-2019
+          runner: windows-latest
           vs_version: '[16, 17)'
         - visualstudio: '2022'
           runner: windows-2022
```

Comment:
Maybe this should also be `windows-latest` for consistency? In the end we don't really care on what windows (or whatever GH makes from it) version this runs. The build tools are the key difference.

Reply:
I thought the intention was to test on different Windows version?  But I'm happy to change them all to `windows-latest`.

Reply:
> I thought the intention was to test on different Windows version?

@dlech was it? Or rather just a byproduct of which image contained which VS version ?